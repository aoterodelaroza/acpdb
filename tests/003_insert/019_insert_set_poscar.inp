## check: 019_insert_set_poscar.out -a1e-10
## delete: 019_insert_set_poscar.db
## labels: regression quick

verbose
system rm -f 019_insert_set_poscar.db
connect 019_insert_set_poscar.db

insert litref bibtex ../zz_source/bsip2.bib
insert set s22x5
 description A description.
 litrefs bsip1 boys1970
 xyz ../zz_source/s22_xyz/ch4_ch4.xyz ../zz_source/s22_xyz/ch4_ch4_1.xyz \
     ../zz_source/s22_xyz/ch4_ch4_2.xyz
 poscar ../zz_source/x23_expt/urea.POSCAR ../zz_source/x23_expt/imidazole.POSCAR
end
