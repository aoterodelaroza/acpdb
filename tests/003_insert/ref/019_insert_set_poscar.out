* SYSTEM: rm -f 019_insert_set_poscar.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 019_insert_set_poscar.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)
# INSERT LITREF bsip1
# INSERT LITREF boys1970
# INSERT LITREF s30l
# INSERT LITREF s12lb
# INSERT LITREF s12la
# INSERT LITREF risthaus2013
# INSERT LITREF xdmlarge
# INSERT LITREF johnson2009
# INSERT LITREF burns2017
# INSERT LITREF parker2015
# INSERT LITREF smith2016
# INSERT LITREF thanthiriwatte2010
# INSERT LITREF marshall2011
# INSERT LITREF burns2011
# INSERT LITREF bauza2013
# INSERT LITREF johnson2003
# INSERT LITREF wilke2009
# INSERT LITREF zhao2005
# INSERT LITREF zhao2005b
# INSERT LITREF grimme2007
# INSERT LITREF parthiban2001
# INSERT LITREF karton2011
# INSERT LITREF kozuch2013
# INSERT LITREF binkley1980
# INSERT LITREF gordon1982
# INSERT LITREF pietro1982
# INSERT LITREF dobbs1987a
# INSERT LITREF dobbs1987b
# INSERT LITREF dithcfield1971
# INSERT LITREF hehre1972
# INSERT LITREF hariharan1973
# INSERT LITREF hariharan1974
# INSERT LITREF gordon1980
# INSERT LITREF francl1982
# INSERT LITREF weigend2005
# INSERT LITREF weigend2006
# INSERT LITREF sto3g
# INSERT LITREF mini
# INSERT LITREF jensen1
# INSERT LITREF jensen2
# INSERT LITREF jensen3
# INSERT LITREF jensen4
# INSERT LITREF b3
# INSERT LITREF lyp
# INSERT LITREF dunning1
# INSERT LITREF dunning2
# INSERT LITREF grimme2010
# INSERT LITREF xdmbasis
# INSERT LITREF witte2016
# INSERT LITREF davidson1986
# INSERT LITREF andzelm1984
# INSERT LITREF kruse2012
# INSERT LITREF hbs66
# INSERT LITREF halogen
# INSERT LITREF dcacp
# INSERT LITREF qcp
# INSERT LITREF hfacp
# INSERT LITREF lasso
# INSERT LITREF osborne2000
# INSERT LITREF wb97xd1
# INSERT LITREF wb97xd2
# INSERT LITREF pepconf
# INSERT LITREF xdmhybrid
# INSERT LITREF xdm
# INSERT LITREF lcwpbe1
# INSERT LITREF lcwpbe2
# INSERT LITREF kalescky2014
# INSERT LITREF nbody1
# INSERT LITREF wateracp
# INSERT LITREF hf-3c
# INSERT LITREF kulik2016
# INSERT LITREF sure2015
# INSERT LITREF goerigk2013
# INSERT LITREF galano2006
# INSERT LITREF jensen2010
# INSERT LITREF hehre1970
# INSERT LITREF curtiss1999
# INSERT LITREF dill1975
# INSERT LITREF clark1983
# INSERT LITREF frisch1984
# INSERT LITREF rassolov1998
# INSERT LITREF mackie2011
# INSERT LITREF burns2013
# INSERT LITREF rezac2011
# INSERT LITREF dilabio2008
# INSERT LITREF mackie2008
# INSERT LITREF mackie2010
# INSERT LITREF torres2012
# INSERT LITREF dilabio2014
# INSERT LITREF vansanten2015
# INSERT LITREF goerigk2011
# INSERT LITREF aorss3
# INSERT LITREF aorss1
# INSERT LITREF aspirin
# INSERT LITREF clathrates
# INSERT LITREF aupi2
# INSERT LITREF pyricarbamate
# INSERT LITREF tmcomplex
# INSERT LITREF wallach
# INSERT LITREF junctions
# INSERT LITREF hb-piperidine
# INSERT LITREF 2nitrile
# INSERT LITREF chpi-pdxanthate
# INSERT LITREF digold
# INSERT LITREF abo3
# INSERT LITREF bipodal-carbamate
# INSERT LITREF oscil-friction
# INSERT LITREF tibond
# INSERT LITREF clathrate-hpr
# INSERT LITREF epxars-fepxas
# INSERT LITREF graphitestress-theo
# INSERT LITREF agtao3surf
# INSERT LITREF xdmchiral
# INSERT LITREF graphenemesa
# INSERT LITREF anales
# INSERT LITREF electrides
# INSERT LITREF xdmgold
# INSERT LITREF kevan
# INSERT LITREF burrito
# INSERT LITREF step
# INSERT LITREF agtao3
# INSERT LITREF xdmc9
# INSERT LITREF kaolin
# INSERT LITREF xdmbench
# INSERT LITREF ncisolids
# INSERT LITREF graphitestress
# INSERT LITREF xdmsolids
# INSERT LITREF energy-correction
# INSERT LITREF cij-partition
# INSERT LITREF supramolecular
# INSERT LITREF qhamgo
# INSERT LITREF ctc-gibbs
# INSERT LITREF gibbs2b
# INSERT LITREF gibbs2a
# INSERT LITREF sbasga2
# INSERT LITREF sbasga2-sbpga2
# INSERT LITREF aggase2
# INSERT LITREF cdal2se4
# INSERT LITREF lapla
# INSERT LITREF qtree
# INSERT LITREF variational
# INSERT LITREF sbpga2
# INSERT LITREF critic
# INSERT LITREF runwien
# INSERT LITREF b88
# INSERT LITREF s225
# INSERT LITREF s22
# INSERT LITREF s66
# INSERT LITREF s668
# INSERT LITREF g3
# INSERT LITREF s22b
# INSERT LITREF s66b
# INSERT LITREF kb49b
# INSERT LITREF kb49a
# INSERT LITREF p26
# INSERT LITREF l7
# INSERT LITREF hsg0
# INSERT LITREF ionichb
# INSERT LITREF gmtkn55
# INSERT LITREF gmtkn30
# INSERT LITREF bsr36
# INSERT LITREF pconf
# INSERT LITREF sconf
# INSERT LITREF water
# INSERT LITREF x40
# INSERT LITREF s8
# INSERT LITREF bde03
# INSERT LITREF aconf
# INSERT LITREF bhperi1
# INSERT LITREF bhperi2
# INSERT LITREF bhperi3
# INSERT LITREF bhperi4
# INSERT LITREF darc
# INSERT LITREF idisp
# INSERT LITREF s668b
# INSERT LITREF adim6
# INSERT LITREF a24
# INSERT LITREF karlsruhe1
# INSERT LITREF karlsruhe2
# INSERT LITREF pople1
# INSERT LITREF pople2
# INSERT LITREF pople3
# INSERT LITREF pople4
# INSERT LITREF pople5
# INSERT LITREF pople6
# INSERT LITREF pople7
# INSERT LITREF pople8
# INSERT LITREF bonifacic1974
# INSERT LITREF brauer2014
# INSERT LITREF clementi1982
# INSERT LITREF stevens1984
# INSERT LITREF topiol1980
# INSERT LITREF topiol1981
# INSERT LITREF duijneveldt1994
# INSERT LITREF davidson1994
# INSERT LITREF xdmpoly
# INSERT LITREF critic2
# INSERT LITREF kahn1972
# INSERT LITREF kahn1976
# INSERT LITREF haynes2006
# INSERT LITREF faver2011
# INSERT LITREF faver2012
# INSERT LITREF jensen2013
# INSERT LITREF goerigk2014
# INSERT LITREF cao2011
# INSERT LITREF dolg2011
# INSERT LITREF kulik2016b
# INSERT LITREF mentel2013
# INSERT LITREF mao2016
# INSERT LITREF deng2011
# INSERT LITREF gutowski1993
# INSERT LITREF xdmsurface
# INSERT LITREF carter2014
# INSERT LITREF linear1
# INSERT LITREF linear2
# INSERT LITREF linear3
# INSERT LITREF spackman2001
# INSERT LITREF linear4
# INSERT LITREF whitten1973
# INSERT LITREF dunlap1979
# INSERT LITREF feyereisen1993
# INSERT LITREF vahtras1993
# INSERT LITREF weigend2002
# INSERT LITREF sodt2006
# INSERT LITREF gpu1
# INSERT LITREF gpu2
# INSERT LITREF gpu3
# INSERT LITREF liechtenstein1995
# INSERT LITREF dudarev1998
# INSERT LITREF kabsch1976
# INSERT LITREF kaminsky2007
# INSERT LITREF maier2015
# INSERT LITREF hornak2006
# INSERT LITREF bsip2

* INSERT: insert data into the database (SET)
# INSERT SET s22x5
# INSERT STRUCTURE s22x5.ch4_ch4
# INSERT STRUCTURE s22x5.ch4_ch4_1
# INSERT STRUCTURE s22x5.ch4_ch4_2
# INSERT STRUCTURE s22x5.urea
# INSERT STRUCTURE s22x5.imidazole

